### PR TITLE
LibWeb: Respond to 'initial', 'inherit' and 'unset' in StyleProperties

### PR DESCRIPTION
**Pull request by AtkinsSJ**
This adds initial values to the generated `PropertyID.h/cpp` files, which are then used by `StyleProperties::property()`. `initial` looks up the initial value, as does `unset` for non-inherited properties. There's a stub for doing an inheritance lookup for the other cases, but actually implementing that (and the cascade in general) is a very big hairy yak, so trying to keep this simple for now.

Weirdly, this does cause a regression for Acid2 - the eyes disappear. My assumption is this is related to an em-sized box getting a size of 0, which is a problem I've seen before with ems, and I don't have a fix for.

We still have `InitialValues` which are set separately in code. Ideally that would disappear, and maybe most of `ComputedValues` could be generated from `Properties.json` too, but I don't have a plan for how to do that yet.

Also, there's a test page for the 3 builtin values now, but it's hard to test them in a way that proves they work. eg, setting `font-weight: initial` has the same effect as `font-weight: green` in our system at the moment, which is tricky.

### CODE REVIEW

File: Userland/Libraries/LibWeb/CSS/Properties.json [hunk 1]
```diff
@@ -72,11 +72,11 @@
     "inherited": false
   },
   "border-bottom-left-radius": {
-    "initial": 0,
+    "initial": "0",
```
Comment (alimpfard) [inline Userland/Libraries/LibWeb/CSS/Properties.json:75]: Perhaps explain why this change is needed (or useful) in the commit description?

File: Userland/Libraries/LibWeb/CSS/StyleValue.h [hunk 2]
```diff
@@ -534,7 +534,11 @@ class CalculatedStyleValue : public StyleValue {
 
 class InitialStyleValue final : public StyleValue {
 public:
-    static NonnullRefPtr<InitialStyleValue> create() { return adopt_ref(*new InitialStyleValue); }
+    static NonnullRefPtr<InitialStyleValue> create()
+    {
+        static NonnullRefPtr<InitialStyleValue> instance = adopt_ref(*new InitialStyleValue);
+        return instance;
```
[the remark this answers is not in the document]
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/CSS/StyleValue.h:542]: My thinking was, keeping the API the same for all StyleValues means outside code doesn't have to know or care about the single-instance thing. I'm thinking at some point it'd make sense to cache other commonly-used values like `0` and `auto`, and `::create()` would silently do that too, though that's making assumptions.

Which of `the()` and `Singleton` is preferable? I've not dealt with either so far.

File: Userland/Libraries/LibWeb/CSS/Properties.json (hunk 1 above)
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/CSS/Properties.json:75]: :+1:

File: Userland/Libraries/LibWeb/CSS/StyleValue.h (hunk 2 above)
[the remark this answers is not in the document]
Comment (alimpfard) [reply Userland/Libraries/LibWeb/CSS/StyleValue.h:542]: I think for this case, `the()` would be just fine.

[the remark this answers is not in the document]
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/CSS/StyleValue.h:542]: :+1:

**Review by alimpfard — APPROVED** [review]
LGTM.
